### PR TITLE
Bump Microsoft.Data.SqlClient and SQLitePCLRaw

### PR DESCRIPTION
**Pull request by bricelam**
Dependabot struggles with prerelease versions...

### CODE REVIEW

**Review by roji — APPROVED** [review]

File: NuGet.config [hunk 1]
```diff
@@ -15,6 +15,8 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <!-- TODO: Remove after mirroring packages -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (bricelam) [inline NuGet.config:19]: @bricelam 🫵 Don't forget.

Comment (AndriySvyryd) [reply NuGet.config:19]: ... or the compliance bots will get you

Comment (roji) [reply NuGet.config:19]: :robot: :boom:

Comment (bricelam) [reply NuGet.config:19]: 🤔 They weren’t sent back in time to attack my mother. Does that mean I’m *guaranteed* to remember…

Comment (roji) [reply NuGet.config:19]: That depends on whether we're using Terminator or Back to the Future rules for time travel...

**Pull request merged**